lib/findAccessorMinMax.js

```diff
@@ -35,15 +34,16 @@ function findAccessorMinMax(gltf, accessor) {
         if (defined(bufferId) && defined(buffers) && buffers.hasOwnProperty(bufferId)) {
             var buffer = buffers[bufferId];
             var source = buffer.extras._pipeline.source;
+
             var count = accessor.count;
             var byteStride = getAccessorByteStride(accessor);
             var byteOffset = accessor.byteOffset + bufferView.byteOffset;
             var componentType = accessor.componentType;
-            var componentByteLength = byteLengthForComponentType(componentType);
 
             for (var i = 0; i < count; i++) {
+                var typedArray = ComponentDatatype.createArrayBufferView(componentType, source.buffer, byteOffset, numberOfComponents);
```

Comment:
Ah good points, I forgot about interleaved accessors. I know it's just a view, but it just seems like a heavy operation to be doing on a per element basis. But if there's now way to avoid it then that's fine.